app/models/game.rb

```diff
@@ -14,6 +14,30 @@ def populate!
     populate_right_white_half!
   end
 
+  def white_check?
```

Comment:
Couldn't we combine these two methods into `check?`, then just accept an argument for the color?